### ISSUE TITLE
Organise markers for the arena

### ISSUE THREAD

**Issue by RealOrangeOne** [srobo/tasks#755]
The arena (and, possibly, things inside it) is marked with libkoki markers, which need printing off in advance.
For the arena itself SR has some pre-printed libkoki markers on a plastic backing which are re-used each year. These need checking and ensuring they make it to the venue. Replacements may be needed for any markers which are showing wear.

### Original

[comp/game/markers](https://github.com/srobo/recurring-tasks/blob/master/comp/game/markers.yaml)

**Comment by Tyler-Ward** [comment]
@RealOrangeOne Do the new markers have a name as we are no longer using libkoki. I expect the printed labels are also no longer valid but am not sure

**Comment by RealOrangeOne** [comment]
Correct, the markers we have printed are no longer valid. This year, we're using "AprilTag" markers (specifically 36H11). I'll update the description.

In the past we discussed using thick (~300gsm) card instead of the previous plastic-based printing to save on costs, logistics and changes, on the basis that repairing damage will be very simple, assuming we can reliably print on the card.

**Comment by PeterJCLaw** [comment]
> assuming we can reliably print on the card

I think we should assume that the inkjet printer we currently have (and referred to in #722) will _not_ be able to print on card (at the least, it's A4 only, may be busy and likely unable to print things with any speed).

**Comment by RealOrangeOne** [comment]
I have contacted a local printing company, on the off chance this is something they can help out with.

**Comment by RealOrangeOne** [comment]
Dan found https://doxdirect.com/, which look to be able to print what we need very easily. 2 complete sets of markers comes in under budget, and on 300gsm card, which is stronger than our original card plan.


<details>
<summary>
Quote
</summary>

![image](https://user-images.githubusercontent.com/6527489/161038683-aa119e8d-85be-43a4-b862-e66e09277c6d.png)

</details>

Unless we have any better leads, I suggest we just go for this.

**Comment by RealOrangeOne** [comment]
We also need markers for the test arena.

We should use dev markers for this, so we don't need competition USBs.

These don't need to be as sturdy, so we can use a lighter card, eg 160gsm


<details>
<summary>
Quote
</summary>


![image](https://user-images.githubusercontent.com/6527489/161046024-0fd99ca6-842e-41bb-ae79-649df9858437.png)

</details>

This sadly pushes us over budget, but not by much, and we can likely keep all sets for future competitions.

**Comment by WillB97** [comment]
2 sets of comp markers (to allow for spares) and a set of dev markers can be ordered as a single order.

![Screenshot_20220331-170815.png](https://user-images.githubusercontent.com/10937272/161101069-fe4cc165-e33b-4c2c-8462-c5a3173737b4.png)

Can we make sure to put a centerline marker above each marker to make them easier to install.

**Comment by RealOrangeOne** [comment]
I'll do you 1 better: Here's 4 center lines https://github.com/RealOrangeOne/zoloto/pull/297

**Comment by RealOrangeOne** [comment]
Order approved [in Slack](https://studentrobotics.slack.com/archives/CMQ49PXPG/p1648742977909669?thread_ts=1648725617.889909&cid=CMQ49PXPG). Will purchase soon.

**Comment by RealOrangeOne** [comment]
2 sets of competition markers and 1 set of dev markers purchased. ETA Monday 11th
